includes/migrate.php

```diff
@@ -167,3 +177,46 @@ function render_constant_empty_notice() {
 
 <?php // phpcs:enable Generic.WhiteSpace.ScopeIndent.IncorrectExact
 }
+
+/**
+ * Iterate through a WordPress Multisite installation and migrate each site.
+ *
+ * @return int The number of sites that were migrated.
+ */
+function migrate_multisite() {
+	$api      = TemplateTags\api();
+	$migrated = 0;
+
+	// Abort early if this isn't a multisite instance.
+	if ( ! is_multisite() ) {
+		return $migrated;
+	}
+
+	$plugin = plugin_basename( dirname( __DIR__ ) . '/wp101.php' );
+	$blogs  = get_sites(
```

Comment:
In case a network has more than 100 sites, this will only process the first 100 (default):

```
# https://developer.wordpress.org/reference/functions/get_sites/
'number'
(int) Maximum number of sites to retrieve. Default 100.
```

Reply:
Great catch, I've added some batching to the process!